Microsoft.Toolkit.Uwp.UI.Controls/ImageEx/ImageEx.Source.cs

```diff
@@ -106,6 +106,14 @@ private static bool IsHttpUri(Uri uri)
                 if (IsCacheEnabled && _isHttpSource)
                 {
                     _image.Source = await ImageCache.GetFromCacheAsync(_uri);
+                    if (_image.Source != null)
```

Comment:
Yes, but I have no idea how to do this, since ExceptionRoutedEventArgs does not seems to have public constructor.